docs/samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v1.yaml

```diff
@@ -0,0 +1,289 @@
+apiVersion: o2ims.provisioning.oran.org/v1alpha1
+kind: ClusterTemplate
+metadata:
+  name: sno-ran-du.v4-Y-Z-1
+  namespace: sno-ran-du-v4-16
+spec:
+  name: sno-ran-du
+  version: v4-Y-Z-1
+  templates:
+    hwTemplate: placeholder-du-template-configmap-v1
+    clusterInstanceDefaults: clusterinstance-defaults-v1
+    policyTemplateDefaults: policytemplate-defaults-v1
+  templateParameterSchema:
+    properties:
+      nodeClusterName:
+        type: string
+      oCloudSiteId:
+        type: string
+      policyTemplateParameters:
```

Comment:
Could we also add "type: object" for policyTemplateParameters? This will help with the input validation to ensure the input for policyTemplateParameters is key-value pairs.